README.md

```diff
@@ -1,111 +1,170 @@
+Find out more about Sendbird Calls for iOS on [Calls SDK for iOS doc](https://sendbird.com/docs/calls/v1/ios/getting-started/about-calls-sdk).
+
+<br />
+
+## Before getting started
+
+This section shows the prerequisites you need to check to use Sendbird Calls SDK for iOS.
+
+### Requirements
 
-## SDK Prerequisites
 * iOS 9.0 or later
 * Swift 4 or later, Objective-C
 * Xcode 9 or later, macOS Sierra or later.
 * Installation of **[Git Large File Storage](https://git-lfs.github.com)**
 
-> **IMPORTANT**: **MAKE SURE to install Git LFS before installing the pod**. Check the size of `WebRTC.framework` in SendBirdWebRTC folder. It MUST be over 800 MB. If the loaded SendBirdWebRTC framework is smaller than that, check the *Git Large File Storage* settings and download again. Refer to [the troubleshooting section](#Library-not-loaded-WebRTC.framework).
+> **IMPORTANT**: **MAKE sure to install Git LFS before installing the pod**. The size of `WebRTC.framework` in **SendBirdWebRTC** folder must be over 800 MB. If the size of the loaded `SendbirdWebRTC` framework is smaller than 800 MB, check the **Git Large File Storage** settings and download again. For further details, refer to SDK’s [troubleshooting section](#library-not-loaded-webrtcframework).
+
+### SDK dependencies
+
+* [WebRTC framework](https://github.com/sendbird/sendbird-webrtc-ios), which can be integrated by `CocoaPods`, `Carthage`, or manual set-up.
+
+<br />
+
+## Getting started
 
-## SDK Dependencies
-* [WebRTC framework](https://github.com/sendbird/sendbird-webrtc-ios), which can be integrated by CocoaPods, Carthage, or direct install.
+This section gives you information you need to get started with Sendbird Calls SDK for iOS.
+
+### Install Calls SDK
+
+To use Sendbird Calls, first add our custom-built `WebRTC` framework to the project. [Git Large File Storage](https://git-lfs.github.com) must be installed to use the `WebRTC `framework along with the SendBirdCalls framework.
 
-## Installation
-To use SendBird Calls, first add our custom-built WebRTC framework to the project. [Git Large File Storage](https://git-lfs.github.com) must be installed to use the WebRTC framework along with the SendBirdCalls framework.
 - Run `brew install git-lfs` in the project directory. 
 
 ### CocoaPods
-[CocoaPods](https://cocoapods.org/) is a dependency manager for Cocoa projects. For usage and installation CocoaPods instructions, visit [their website](https://cocoapods.org/). SendBirdCalls can be integrated into the Xcode project using CocoaPods by doing the following:
-1. Install CocoaPods into the project by running `pod init` in the project directory.
+
+[CocoaPods](https://cocoapods.org/) is a dependency manager for Cocoa projects. The `SendbirdCalls` framework can be integrated into the Xcode project using `CocoaPods` by doing the following.
```

Comment:
`SendbirdCalls` should have the `B` capitalized as well.

Reply:
I see few other `SendbirdCalls` without `B` capitalization; can you look at them as well? 🙏

Reply:
Done, all are checked!